gateway/src/main/java/io/zeebe/gateway/impl/job/LongPollingActivateJobsHandler.java

```diff
@@ -156,9 +160,15 @@ private void block(JobTypeAvailabilityState state, LongPollingActivateJobsReques
 
   private void addTimeOut(JobTypeAvailabilityState state, LongPollingActivateJobsRequest request) {
     ActorClock.currentTimeMillis();
+    final Duration requestTimeout;
+    if (request.useDefaultTimeout()) {
+      requestTimeout = longPollingTimeout;
+    } else {
+      requestTimeout = request.getLongPollingTimeout();
+    }
```

Comment:
What do you think about setting the default timeout on the creation of the request? So, we don't need to deal with `null` or default values afterward.

Reply:
Updated as we discussed :+1: